XML::Simple is not actually required until runtime; allows cross-compiling

--- a/configure.ac
+++ b/configure.ac
@@ -16,12 +16,6 @@ fi
 if test -z "`$PERL -v | fgrep 'v5.' 2> /dev/null`" -a -z "`$PERL -v | fgrep 'version 5.' 2> /dev/null`"; then
    AC_MSG_ERROR([perl 5.x required for icon-naming-utils])
 fi
-AC_MSG_CHECKING([for XML::Simple])
-if `$PERL -e "require XML::Simple" 2>/dev/null`; then
-   AC_MSG_RESULT([ok])
-else
-   AC_MSG_ERROR([XML::Simple perl module is required for icon-naming-utils])
-fi
 
 AC_CONFIG_FILES([
 Makefile
